app/assets/javascripts/admin/analytics-modules/ga4-link-setup.js

```diff
@@ -16,10 +16,10 @@ window.GOVUK.analyticsGa4.analyticsModules =
             event_name: 'navigation',
             type: link.role === 'button' ? 'button' : 'generic_link'
           }
-          if (link.dataset.ga4Event) {
-            Object.assign(event, JSON.parse(link.dataset.ga4Event))
+          if (link.dataset.ga4Link) {
```

Comment:
Worth linking to https://github.com/alphagov/govuk_publishing_components/blob/2bf9077e3e091a2151feab56a715035e1883c689/app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js#L9 in the commit message to more clearly show why this is being updated?